shark_turbine/kernel/wave/visualization.py

```diff
@@ -0,0 +1,19 @@
+disabled_graphviz = False
+try:
+    import pygraphviz as pgv
+except:
+    disabled_graphviz = True
+from torch import fx
+
+
+def visualize_graph(graph: fx.Graph, file_name: str):
+    if disabled_graphviz:
+        return
```

Comment:
Good point. So the motivation for disabled_graphviz was on some of the cluster machines, I did not have sudo privileges so could not install graphviz (which is required for pygraphviz). On a machine with no graphviz, the question is what is the appropriate behavior for tests that call this function? Right now it would fail silently, but I agree it's probably better to throw some notification indicating the failure. Instead of erroring out and interrupting the program, how about a warning instead?

Reply:
Sure, will change.